cmd/formula-analytics.rb

```diff
@@ -229,7 +276,15 @@ def influx_analytics(args)
           rescue ArgumentError, TypeError
             nil
           end
+
+          next if count <= 0
+
+          count
         end
+
+        # TODO: we don't seem to get a valid count for these categories, unclear why.
+        count ||= 1 if count_being_weird_categories.include?(category)
```

Comment:
@SMillerDev @Bo98 (and any other @homebrew/maintainers who understand InfluxDB): similarly: it'd be good to figure this out. I'm actually not convinced the counts are correct in this case (so am not going to have the data displayed publicly anywhere yet) but hopefully this being merged gives you enough to reproduce the weirdness with some `puts` or `pp`.

Reply:
@Bo98 Ah, makes sense now, thanks. Yeh, I think moving this to SQL would be desirable on multiple counts.